src/controller/tests/data_model/BUILD.gn

```diff
@@ -40,4 +40,4 @@ chip_test_suite_using_nltest("data_model") {
     "${chip_root}/src/transport/raw/tests:helpers",
     "${nlunit_test_root}:nlunit-test",
   ]
-}
+}
```

Comment:
some of these files seem to have lost the trailing newline. Could we add it back?